cargo-pgx/src/command/schema.rs

```diff
@@ -555,7 +555,7 @@ fn create_stub(
     so_rustc_invocation.stderr(Stdio::inherit());
 
     if let Some(rustc_flags_str) = std::env::var("RUSTFLAGS").ok() {
-        let rustc_flags = rustc_flags_str.split(" ").collect::<Vec<_>>();
+        let rustc_flags = rustc_flags_str.split(' ').collect::<Vec<_>>();
```

Comment:
I feel doubtful that this isn't monomorphized away by LLVM, but I've noticed subtle differences in these cases, so very possible still.